libpod/container_inspect.go

```diff
@@ -257,6 +257,26 @@ func (c *Container) GetInspectMounts(namedVolumes []*ContainerNamedVolume, image
 	return inspectMounts, nil
 }
 
+// GetSecurityOptions retrives and returns the security related annotations and process information upon inspection
+func (c *Container) GetSecurityOptions(ctrSpec *spec.Spec) []string {
```

Comment:
Why does ctrSpec need to be an argument - it's stored as `c.config.Spec`?

Reply:
fixed